src/mongo/dbtests/updatetests.cpp

```diff
@@ -58,6 +61,11 @@ class ClientBase {
 public:
     ClientBase() : _client(&_opCtx) {
         mongo::LastError::get(_opCtx.getClient()).reset();
+        repl::ReplSettings replSettings;
+        replSettings.setOplogSizeBytes(10 * 1024 * 1024);
+        replSettings.setMaster(true);
```

Comment:
This uses a different configuration for the ReplicationCoordinator from the default:

https://github.com/mongodb/mongo/blob/master/src/mongo/dbtests/dbtests.cpp#L155

Reply:
@apocarteres basically, the requested change is to remove the setMaster() call and set the current replica set node as the PRIMARY:

 repl::getGlobalReplicationCoordinator()
        ->setFollowerMode(repl::MemberState::RS_PRIMARY)
        .ignore();

Reply:
@benety thanks. seems that it's better just to save repl settings in "repl" test suite and restore it when "repl" suite is done instead of hacking repl settings in "update" suite